code/modules/clothing/suits/labcoat.dm

```diff
@@ -30,34 +56,22 @@
 /obj/item/clothing/suit/storage/toggle/labcoat/mad
 	name = "The Mad's labcoat"
```

Comment:
This is such a bad name. What is this even for?

Reply:
Apparently dates back to before 2011. Primordial code.